content/community/blm.rst

```diff
@@ -0,0 +1,149 @@
+      or anyone they list on their donation page.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://blacklivesmatters.carrd.co/#donate'
+        :content: 'Donate through Black Lives Matter'
+
+  - title: Black Visions Collective
+    url: 'https://www.blackvisionsmn.org/'
+    icon: 'fist'
+    text: |
+      Black Visions Collective believes in a future
+      where all Black people have autonomy,
+      safety is community-led,
+      and we are in right relationship within our ecosystems.
+      BLVC is committed to a long term vision in which
+      ALL Black lives not only matter, but are able to thrive.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://secure.everyaction.com/4omQDAR0oUiUagTu0EG-Ig2'
+        :content: 'Donate to Black Visions Collective'
+
+  - title: Southern Poverty Law Center
+    url: 'https://www.splcenter.org/'
+    icon: 'fist'
+    text: |
+      The SPLC is dedicated to fighting hate and bigotry
+      and to seeking justice
+      for the most vulnerable members of our society.
+      Using litigation, education,
+      and other forms of advocacy,
+      the SPLC works toward the day when the ideals of equal justice
+      and equal opportunity will be a reality.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://donate.splcenter.org/'
+        :content: 'Donate to Southern Poverty Law Center'
+
+  - title: Innocence Project
+    url: 'https://www.innocenceproject.org/'
+    icon: 'fist'
+    text: |
+      The Innocence Project's mission
+      is to free the staggering number of innocent people
+      who remain incarcerated,
+      and to bring reform to the system
+      responsible for their unjust imprisonment.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://www.innocenceproject.org/donate/'
+        :content: 'Donate to the Innocence Project'
+
+  - title: Campaign Zero
+    url: 'https://www.joincampaignzero.org/'
+    icon: 'fist'
+    text: |
+      Funds donated to Campaign Zero
+      support the analysis of policing practices across the country,
+      research to identify effective solutions to end police violence,
+      technical assistance to organizers
+      leading police accountability campaigns
+      and the development of model legislation and advocacy
+      to end police violence nationwide.
+
+      .. callmacro:: content.macros.j2#link_button
+        :url: 'https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ZHQKRGV7JHCWC&source=url'
+        :content: 'Donate to Campaign Zero'
+
+  - title: Black Tech for Black Lives
+    url: https://www.blacktechforblacklives.com/
+    icon: 'fist'
+    text: |
+      "#BlackTech4BlackLives is a collective of Black tech entrepreneurs,
+      investors, creatives, changemakers, and workers,
+      united to use our social, political,
+      and economic capital for the advancement of our communities."
```

Comment:
This is the only thing on this page that I wondered if it was intentional since none of the other organizations had quoted content.